cmd/libs/go2idl/args/args.go

```diff
@@ -71,11 +74,14 @@ type GeneratorArgs struct {
 }
 
 func (g *GeneratorArgs) AddFlags(fs *pflag.FlagSet) {
+	//FIXME: can we simplify operation to one input file, one output?
```

Comment:
I realized even deep_copy doesn't start with a root file and only process symbols linked transitively from there.  I have re-pushed this so that it uses all files in the same dir as deps for a rebuild, except generated files.